src/container/connect-pane-container.tsx

```diff
@@ -7,28 +7,14 @@ export interface ConnectPaneProps {
 }
 
 export const ConnectPaneContainer: React.SFC<ConnectPaneProps> = props => (
-	<div>
-		<Components.Space sizeBottom={Components.SpaceSize.L}>
-			<Components.Headline textColor={Components.colors.grey20} order={2}>
-				Connect to a Library
-			</Components.Headline>
-		</Components.Space>
-		<Components.Space sizeBottom={Components.SpaceSize.XXXL}>
-			<Components.Copy size={Components.CopySize.M} textColor={Components.colors.grey20}>
-				To build prototypes with already existing components, connect to your React component
-				library first.
-			</Components.Copy>
-		</Components.Space>
-		<Components.Space sizeBottom={Components.SpaceSize.S}>
-			<Components.Button
-				onClick={props.onPrimaryButtonClick}
-				order={Components.ButtonOrder.Primary}
-			>
-				Connect
-			</Components.Button>
-		</Components.Space>
-		<Components.Link color={Components.colors.grey50} onClick={props.onSecondaryButtonClick}>
-			<Components.Copy size={Components.CopySize.S}>Open Alva Example</Components.Copy>
-		</Components.Link>
-	</div>
+	<ConnectLibrary
+		headline={'Connect to a Library'}
```

Comment:
Using direct JSX notation might be a bit easier on the eyes: 

`headline="Connect to a Library"` etc